HDSAviz/Basic_Plot_Updated.py

```diff
@@ -0,0 +1,130 @@
+                **Note file must be in same folder as script.**}
+                Delete upon change
+    Cols: Names of the columns to be plotted (Array of strings)
+    minvalues: Cutoff minimum for which parameters should be plotted (Array of
+                floats)
+    Returns:
+    p: Figure of the data to be plotted
+    """
+    # Read in csv file as panda dataframe.
+    tdf = pd.read_csv(Dataframe, delimiter=' ', skipinitialspace=True,
+                      engine='python')
+    df = tdf
+    # df = Dataframe
+    maxval = 0
+    # Remove rows which have values less than cutoff values
+    for i in range(0, minvalues.size):
+        df = df[df[Cols[i]] > minvalues[i]]
+        if maxval < max(df[Cols[i]]):
+            maxval = max(df[Cols[i]])
+    df = df.dropna()
+
+    # Create dictionary of bar colors, assume up to 3 stats are plotted
+    # per parameter.
+    colors = ["#0d3362", "#c64737", "black"]
+    stat_color = OrderedDict()
+    for i in range(0, Cols.size):
+        stat_color[i] = colors[i]
+    # Reset index of dataframe.
+    df = df.reset_index(drop=True)
+
+    # Sizing parameters
+    width = 800
+    height = 800
+    inner_radius = 90
+    outer_radius = 300 - 10
+
+    # Determine wedge size based off number of parameters
+    big_angle = 2.0 * np.pi / (len(df)+1)
+    # Determine division of wedges for plotting bars based on # stats plotted
+    small_angle = big_angle / (minvalues.size+2)
+
+    p = figure(plot_width=width, plot_height=height, title="",
+               x_axis_type=None, y_axis_type=None,
+               x_range=(-420, 420), y_range=(-420, 420),
+               min_border=0, outline_line_color="black",
+               background_fill_color="#f0e1d2", border_fill_color="#f0e1d2")
+
+    p.xgrid.grid_line_color = None
+    p.ygrid.grid_line_color = None
+
+    # annular wedges divided into smaller sections for bars
+    angles = np.pi/2 - big_angle/2 - df.index.to_series()*big_angle
+    # circular axes and labels
+    labels = np.power(10.0, np.arange(0, -5, -1))
+    labz = np.power(10.0, np.arange(-0.30103, -4.30103, -1))
+    # Re-size if no values are above 0.1
+    if maxval < 0.1:
+        labels = np.delete(labels, 0)
+    # Size of radii of each label
+    # pdb.set_trace()
+    rad = (((np.log10(labz / labels[0])) + labels.size) *
+           (outer_radius - inner_radius) / labels.size + inner_radius)
+    radii = (((np.log10(labels / labels[0])) + labels.size) *
+             (outer_radius - inner_radius) / labels.size + inner_radius)
+    # Add zero label to labels and radii
+    labels = np.append(labels, 0.0)
+    radii = np.append(radii, inner_radius)
+    rad = np.append(rad, radii)
+    p.circle(0, 0, radius=rad, fill_color=None, line_color="white")
+    p.text(0, radii[:], [str(r) for r in labels[:]],
+           text_font_size="8pt", text_align="center", text_baseline="middle")
+    # Plot the values of each stat for each parameter.
+    for c in range(0, Cols.size):
```

Comment:
perhaps just a comment above these rad assignments is sufficient; if the variable names get too long that adds its own set of problems for readability.